app/models/micropost.rb

```diff
@@ -0,0 +1,17 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  default_scope -> {order created_at: :desc}
```

Comment:
Use scope instead of default scope, as the default scope in rails anti-pattern. 
 Reference: [Default Scopes are an Anti-Pattern](http://weblog.jamisbuck.org/2015/9/19/default-scopes-anti-pattern.html)